sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/_version.py

```diff
@@ -4,6 +4,6 @@
 # Licensed under the MIT License.
 # ------------------------------------
 
-VERSION = "1.1.0"
+VERSION = "1.0.1"
```

Comment:
why is the version going down to `1.0.1`

Reply:
You only bump to a patch release for a deprecation @swathipil

Reply:
The last released version was 1.0.0 @iscai-msft. As per previous guidelines as well, we do a patch bump for deprecation.